app/views/widgets/ad_landing_page/_messaging_with_image.html

```diff
@@ -1,13 +1,42 @@
-<div class="ad-landing-messaging-widget ad-landing-messaging-widget__image ad-landing-messaging-widget__text-left">
+<div class="ad-landing-messaging-widget ad-landing-messaging-widget__image ad-landing-messaging-widget__text-right">
   <div class="outer-container">
     <div class="text-column">
       <div class="section-message">
-        <h3 class="title">Read what one of our graduates has to say:</h3>
+        <h3 class="title">Read what one of our graduates has to say</h3>
         <div class="body editableContent">
           <p>
-            <img class="media" src="http://www.critterbabies.com/wp-content/gallery/kittens/Cute-Kitten-kittens-16122946-1280-800.jpg"/>
-            "I came to Chapman for reason: QUALITY. Quality in education, quality in professors, quality in career services, and quality in my fellow classmates. Chapman is constantly striving for greatness, and they have given me the resources to achieve the same throughout my hourney. Because of the intimate setting Chapman provides, students are able to take full advantage of the resources and services they offer and, most importantly, get to know the faculty, staff, and fellow classmates on a more casual level."<br /><br /><strong>Jackie Becerra, MBA '15</strong><br />Nike, Inc. Intern
+            <img alt="student headshot" class="media" src="/_files/img/ad-landing-150x250.jpg">
```

Comment:
Notice: if you need to include assets like images in Cascade Assets that you don't want bundled up for deployment, this is how you do it: put them in the `static` directory (see image file below).